docs/get-started/install.rst

```diff
@@ -54,7 +54,7 @@ lower-level code.
 ::
 
     sudo apt-get install pigpiod
-    pip install pigpio
+    pip3 install pigpio
```

Comment:
Why not just __pip3 install -r requirements.txt__ ?

Reply:
For the getting started section, every step should be as explicit as possible, for educational purposes (even if a quicker `requirements.txt` is also provided).